run_marqo.sh

```diff
@@ -1,28 +1,44 @@
 #!/bin/bash
-#source /opt/bash-utils/logger.sh
+
+# set the default value to info and convert to lower case
+export MARQO_LOG_LEVEL=${MARQO_LOG_LEVEL:-info}
+MARQO_LOG_LEVEL=`echo "$MARQO_LOG_LEVEL" | tr '[:upper:]' '[:lower:]'`
+
 export PYTHONPATH="${PYTHONPATH}:/app/src/"
 export CUDA_HOME=/usr/local/cuda/
 export LD_LIBRARY_PATH=${CUDA_HOME}/lib64
 export PATH=${CUDA_HOME}/bin:${PATH}
 
 trap "bash /app/scripts/shutdown.sh; exit" SIGTERM SIGINT
-echo "Python packages:"
-pip freeze
+
+if [ "$MARQO_LOG_LEVEL" = "debug" ]; then
+  echo "Python packages:"
+  pip freeze
+fi
 
 function wait_for_process () {
     local max_retries=30
     local n_restarts_before_sigkill=3
     local process_name="$1"
     local retries=0
-    while ! [[ $(docker ps -a | grep CONTAINER) ]] >/dev/null && ((retries < max_retries)); do
-        echo "Process $process_name is not running yet. Retrying in 1 seconds"
-        echo "Retry $retries of a maximum of $max_retries retries"
+    while ! [[ $(docker ps -a | grep CONTAINER) ]] > /dev/null 2>&1 && ((retries < max_retries)); do
+        if [ "$MARQO_LOG_LEVEL" = "debug" ]; then
```

Comment:
Same comment as earlier, if we forsee multiple log levels wanting to output this message, we could make this an inequality, and have a ranking list of log levels somewhere.

Reply:
ditto my reply